source/manage/cloud-team-and-channel.rst

```diff
@@ -23,6 +21,22 @@ Team Profile
 
 The name and description of the team.
 
+Within a Team Profile, System Admins can archive or unarchive the team. Archiving a team makes its contents inaccessible for all users. All related information is archived, including posts from the database. Before archiving a team, we recommend that you perform a database backup.
```

Comment:
```suggestion
System Admins can archive or unarchive the team from within **User Management > Teams > Team Management**. Archiving a team makes its contents inaccessible for all users. All related information is archived, including posts from the database. Before archiving a team, we recommend that you perform a database backup.
```

Reply:
0/5 on this, but thinking it might be easier to understand if context is provided. I almost certainly got the path wrong though.